Dockerfile.dapper

```diff
@@ -0,0 +1,19 @@
+FROM registry.suse.com/bci/golang:1.19
```

Comment:
Can we bump the tag to `1.20` to match the upstream's GO version?

Reply:
updated